.yarnrc.yml

```diff
@@ -8,15 +8,16 @@ immutablePatterns:
 
 initScope: yarnpkg
 
+logFilters:
+  - code: YN0005
+    level: discard
```

Comment:
These messages annoy me to no end and I find them completely useless. I'm happy to remove this filter and only apply it inside my home configuration if anybody finds them useful in any way.

Reply:
Ideally they should only appear once, when the build is truly skipped instead of being run (not each and every time!), but we don't quite have the tech yet. I think I'd prefer to keep them displayed in our repo if only to eventually motivate me to fix that when I have the time 😄